src/terra-dev-site/app/components.2/example/ApplicationStatusOverlayProps.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'terra-button';
+
+/**
+ * The props table parser continues to have trouble detecting the propTypes of a component that
+ * returns `null`. The propType definition is duplicated here so that a table could be parsed for it.
+ */
+
+const propTypes = {
+  /**
+   * An array of objects containing terra-button properties. A key attribute is required for each object.
+   * This array is used to render buttons in the bottom section.
+   * Example:`[{ text: 'Button 1', key: 1, size: 'medium', variant: 'action', onClick: onClickFunction}]`
```

Comment:
Example needs to be updated - there is no longer `size` as a button prop and `action` button should not be recommended or suggested to be used here, only neutral or emphasis.
```suggestion
   * Example:`[{ text: 'Refresh Application', key: 1, variant: 'neutral', onClick: myRefreshFunction}]`
```

Reply:
Changed here : https://github.com/cerner/terra-application/pull/47/commits/62b43a41763ab55e834f07ce82e374ac33e0b409